zulip/integrations/bridge_with_matrix/README.md

```diff
@@ -1,20 +1,19 @@
 # Matrix <--> Zulip bridge
 
-This acts as a bridge between Matrix and Zulip. It also enables a
-Zulip topic to be federated between two Zulip servers.
+This acts as a bridge between Matrix and Zulip.
 
-## Usage
+### Enhanced Features
+- Supporting multiple (Zulip topic, Matrix channel)-pairs.
+- Handling files according to their mimetype.
 
-### For IRC bridges
 
-Matrix has been bridged to the listed
-[IRC networks](https://github.com/matrix-org/matrix-appservice-irc/wiki/Bridged-IRC-networks),
-where the 'Room alias format' refers to the `room_id` for the corresponding IRC channel.
+## Installation
 
-For example, for the freenode channel `#zulip-test`, the `room_id` would be
-`#freenode_#zulip-test:matrix.org`.
+In case you'd like encryption to work, you need pip to install the `matrix-nio`
```

Comment:
Maybe add the obvious, but necessary `pip install -r requirements.txt` before this sentence.

Reply:
sure, thanks!